app/helpers/application_helper.rb

```diff
@@ -175,6 +175,10 @@ def sortable(column, title = nil)
     link_to title, params.except('action', 'controller').permit!.merge(sort: column, direction: direction)
   end
 
+  def mentor_comments_for_application(application)
```

Comment:
Oh sorry, I should have been more clear about the "helper method" bit. I meant to leave it in the controller; basically making the private/protected method that you added available to the view using the `helper_method` macro. We already know the application instance so not having to add it as a functional argument improves readability imho.

Reply:
Ahh! Didn't know that was a thing. 😅 Thank you for the feedback!